src/System.Runtime/tests/System/Reflection/PointerTests.cs

```diff
@@ -103,6 +105,7 @@ public void IntPtrFieldSetValue(int value)
 
         [Theory]
         [MemberData(nameof(Pointers))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "Pointers through Invoke not implemented: https://github.com/dotnet/corert/issues/2113")]
```

Comment:
OK fine. In generally I think it's fine to put a TFS bug number in an open source comment.  I mean it's not ideal but it's all about getting the job done. @weshaggard who objected to this?